### DISCUSSION TITLE
I've unpublished my site, how do I get it back?

### DISCUSSION THREAD

**Discussion by toastonrye** [community/community#55593] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hi, I am trying to learn github. I've unpublished my site, and I can't figure out how to get it back. Everything I have Googled doesn't work, and I don't understand what unpublishing means and why it's so difficult to get it back? I would have expected a republished button right where the unpublish button was.

When I go to actions>pages-build>deployment> rerun all jobs I don't see that button.

https://github.com/orgs/community/discussions/29619#discussioncomment-3692599

**Comment by toastonrye** [comment]
I've just deleted it and remade it.

**Reply by jge162** [reply]
So is it working now?

previous issue was Jekyll theme "just-the-docs" could not be found, which is causing the build process to fail.

**Reply by toastonrye** [reply]
Yes it's working now. When I unpublished the site, I broke something and found it easier to just remake.
